README.md

```diff
@@ -1,12 +1,35 @@
-<img src="images/flagD.png" width="350px;" >
-
-
-![build](https://img.shields.io/github/workflow/status/open-feature/flagd/ci)
-![goversion](https://img.shields.io/github/go-mod/go-version/open-feature/flagd/main)
-![version](https://img.shields.io/badge/version-pre--alpha-green)
-![status](https://img.shields.io/badge/status-not--for--production-red)
-
-Flagd is a simple command line tool for fetching and evaluating feature flags for services. It is designed to conform with the OpenFeature specification.
+<h1 align="center">
+  <img src="images/flagD.png" width="350px;" >
+</h1>
+
+<h2 align="center">A feature flag daemon with a Unix philosophy.</h4>
+
+<p align="center">
+  <a href="https://github.com/open-feature/flagd/actions">
+    <img src="https://github.com/open-feature/flagd/actions/workflows/build.yaml/badge.svg" alt="Github Actions">
+  </a>
+  <a href="https://goreportcard.com/report/github.com/open-feature/flagd">
+    <img src="https://goreportcard.com/badge/github.com/open-feature/flagd">
+  </a>
+  <img src="https://img.shields.io/github/go-mod/go-version/open-feature/flagd">
+  <a href="https://github.com/open-feature/flagd/releases">
+    <img src="https://img.shields.io/github/release/open-feature/flagd/all.svg">
+  </a>
+</p>
+
+## Features
+
+- OpenFeature compliant with providers available in many languages.
```

Comment:
Since these are fragments and not sentences, please remove the capitalization and periods.

Reply:
I can remove the punctuation but it feels weird not to capitalize the bullet points.